Source/Scene/GltfIndexBufferLoader.js

```diff
@@ -338,12 +342,7 @@ GltfIndexBufferLoader.prototype.process = function (frameState) {
     return;
   }
 
-  // WebGL1 has no way to retrieve the contents of buffers that are
-  // on the GPU. Therefore, the index buffer must be stored in CPU memory
-  // to generate wireframes for models.
-  const useWebgl2 = frameState.context.webgl2;
-  const loadTypedArrayForWireframe = !useWebgl2 && this._loadForWireframe;
-  if (this._loadAsTypedArray || loadTypedArrayForWireframe) {
+  if (!this._loadBuffer) {
```

Comment:
We discussed this offline, we decided to reorganize this function a little bit to avoid the duplicate code and early return. See https://github.com/CesiumGS/cesium/pull/10397/files#diff-d58e7af8b44d5b26b11ae14d06508206d2b7a07722ac16bb3c1546fafdef8da8 for how I did it in my PR